core/src/test/scala/integration/kafka/api/ProducerFailureHandlingTest.scala

```diff
@@ -211,16 +227,27 @@ class ProducerFailureHandlingTest extends KafkaServerTestHarness {
     assertThrows(classOf[IllegalStateException], () =>  producer3.send(record))
   }
 
-  @Test
-  def testCannotSendToInternalTopic(): Unit = {
-    TestUtils.createOffsetsTopic(zkClient, servers)
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumName)
+  @ValueSource(strings = Array("zk", "kraft"))
+  def testCannotSendToInternalTopic(quorum: String): Unit = {
+
+    admin = TestUtils.createAdminClient(brokers, listenerName,
+      TestUtils.securityConfigs(Mode.CLIENT,
+        securityProtocol,
+        trustStoreFile,
+        "adminClient",
+        TestUtils.SslCertificateCn,
+        clientSaslProperties))
+    TestUtils.createOffsetsTopicWithAdmin(admin, brokers)
+    //TestUtils.createOffsetsTopic(zkClient, brokers)
```

Comment:
there's a `createOffsetsTopic` defined in `KafkaServerTestHarness` which takes care of creating and deleting an admin client